common/rdr/Exception.cxx

```diff
@@ -49,6 +52,24 @@ Exception::Exception(const char *format, ...) {
 	va_end(ap);
 }
 
+NetworkException::NetworkException(const char* s, int err)
+  : Exception("%s", s)
+{
+  strncat(str_, ": ", len-1-strlen(str_));
+#ifdef _WIN32
+  wchar_t *currStr = new wchar_t[len-strlen(str_)];
+  currStr = gai_strerrorW(err);
+  WideCharToMultiByte(CP_UTF8, 0, currStr, wcslen(currStr), str_+strlen(str_),
+                      len-1-strlen(str_), 0, 0);
+  delete [] currStr;
```

Comment:
Is this your responsibility? What is documented for `gai_strerrorW()` with regard to the lifetime of the returned buffer?